ledger-service/http-json/src/it/scala/http/HttpServiceIntegrationTestUserManagement.scala

```diff
@@ -0,0 +1,162 @@
+      primaryParty: Option[Ref.Party] = None,
+      initialRights: List[UserRight] = List.empty,
+  ): Future[User] =
+    ledgerClient.userManagementClient.createUser(
+      User(userId, primaryParty),
+      initialRights,
+      Some(participantAdminJwt.value),
+    )
+
+  def headersWithUserAuth(userId: String): List[Authorization] =
+    authorizationHeader(jwtForUser(userId))
+
+  def getUniqueUserName(name: String): String = getUniqueParty(name).toString
+
+}
+
+@SuppressWarnings(Array("org.wartremover.warts.NonUnitStatements"))
+class HttpServiceIntegrationTestUserManagementNoAuth
+    extends AsyncFreeSpec
+    with Matchers
+    with Inside
+    with StrictLogging
+    with HttpServiceIntegrationTestUserManagementFuns {
+
+  override def jdbcConfig: Option[JdbcConfig] = None
+
+  override def staticContentConfig: Option[StaticContentConfig] = None
+
+  override def useTls: UseTls = UseTls.Tls
+
+  override def wsConfig: Option[WebsocketConfig] = None
+
+  import scalaz.syntax.tag._
+
+  "create IOU should work with correct user rights" in withHttpServiceAndClient(
+    participantAdminJwt
+  ) { (uri, encoder, _, ledgerClient, _) =>
+    val alice = getUniqueParty("Alice")
+    val command: domain.CreateCommand[v.Record, OptionalPkg] = iouCreateCommand(alice.unwrap)
+    val input: JsValue = encoder.encodeCreateCommand(command).valueOr(e => fail(e.shows))
+    for {
+      user <- createUser(ledgerClient)(
+        Ref.UserId.assertFromString(getUniqueUserName("nice.user")),
+        initialRights = List(
+          CanActAs(Ref.Party.assertFromString(alice.toString))
+        ),
+      )
+      (status, output) <- postJsonRequest(
+        uri.withPath(Uri.Path("/v1/create")),
+        input,
+        headers = headersWithUserAuth(user.id),
+      )
+      assertion <- {
+        status shouldBe StatusCodes.OK
+        assertStatus(output, StatusCodes.OK)
+        val activeContract = getResult(output)
+        assertActiveContract(activeContract)(command, encoder)
+      }
+    } yield assertion
+  }
+
+  "create IOU should fail if user has no permission" in withHttpServiceAndClient(
+    participantAdminJwt
+  ) { (uri, encoder, _, ledgerClient, _) =>
+    val alice = getUniqueParty("Alice")
+    val bob = getUniqueParty("Bob")
+    val command: domain.CreateCommand[v.Record, OptionalPkg] = iouCreateCommand(alice.unwrap)
+    val input: JsValue = encoder.encodeCreateCommand(command).valueOr(e => fail(e.shows))
+    for {
+      user <- createUser(ledgerClient)(
+        Ref.UserId.assertFromString(getUniqueUserName("nice.user")),
+        initialRights = List(
+          CanActAs(Ref.Party.assertFromString(bob.toString))
+        ),
+      )
+      (status, output) <- postJsonRequest(
+        uri.withPath(Uri.Path("/v1/create")),
+        input,
+        headers = headersWithUserAuth(user.id),
+      )
+      assertion <- {
+        status shouldBe StatusCodes.BadRequest
+        assertStatus(output, StatusCodes.BadRequest)
```

Comment:
Why is this a `400`? Shouldn't that be a `403`?

Reply:
This is what I referred to what I will fix later in a follow up PR :sweat:

Reply:
Sounds fine. What I tend to do for known deficiencies is to add a `FIXME: <what's amiss>` to save reviewer's time by making them aware that you're aware of the deficiency ;-)

Reply:
That's a very good idea, thanks! Will add a commit that adds that :)